buzzmobile/process/gps_mapper/gps_mapper.py

```diff
@@ -92,23 +91,18 @@ def update_image():
         point = median_filter(g['location'])
         point = (point[1], -point[0])
         if g['y_range'] is not 0 and point is not None:
-            point = interpolate.normalize_single_point(point, g['top_left'],
-                    g['ll_height'],
-                    g['ll_width'],
-                    g['height'],
-                    g['width'])
+            point = interpolate.normalize_single_point(point,
+                    g['top_left'], g['ll_height'], g['ll_width'],
+                    (g['height'], g['width']))
             # Call method to calculate rotated points and interpolate a path
             # between the points that are in the current window
             # (based on the current location and bearing).
             result = interpolate.window(g['points'], point, g['bearing'],
-                     line_width,
-                     sigma_x,
-                     sigma_y,
-                     image_height,
-                     image_width)
+                     (SIGMA_X, SIGMA_Y),
+                     (IMAGE_HEIGHT, IMAGE_WIDTH, LINE_WIDTH))
```

Comment:
yeah gonna do the namedtuple thing
not really sure how to use fewer globals
i mean i would either use them here or in interpolate.py because i need those numbers